Dockerfile-python

```diff
@@ -1,5 +1,23 @@
-FROM python:3
-RUN pip install postgres
-RUN pip install psycopg2
+FROM python:3.6
+COPY requirements.txt /
+RUN pip install -r /requirements.txt
 RUN apt update
-RUN apt install -y postgresql 
\ No newline at end of file
+RUN apt install -y postgresql 
+
+RUN apt-get update \
```

Comment:
Is there any difference between `apt-get update` & `apt update`. If not, please get rid of extra occurences